File: api/BeHealth.Business/Resources/Appointment/AppointmentResource.cs
using BeHealth.Business.Resources.Clinets;
using BeHealth.Presentence.Enums;
using System;
using System.Collections.Generic;
using System.Text;

namespace BeHealth.Business.Resources.Appointment
{
    public class AppointmentResource
    {
        public long ID { get; set; }

        public DateTimeOffset StartTime { get; set; }
        public DateTimeOffset EndTime { get; set; }

        public AppointmentStatus Status { get; set; }

        public ClientResource Client { get; set; }
    }
}

File: app/src/app/features/home/resources/appointment.resource.ts
import { ClientResource } from "../../clients/resources/client.resource";
import { AppointmentStatus } from "../enums/appointment-status.enum";

export  class AppointmentResource{
    id: number;
    startTime: Date;
    endTime: Date;
    status: AppointmentStatus;
    client: ClientResource
}